homeassistant/components/isy994/button.py

```diff
@@ -23,13 +34,22 @@ async def async_setup_entry(
     """Set up ISY/IoX button from config entry."""
     hass_isy_data = hass.data[ISY994_DOMAIN][config_entry.entry_id]
     isy: ISY = hass_isy_data[ISY994_ISY]
-    uuid = isy.configuration["uuid"]
-    entities: list[ISYNodeQueryButtonEntity | ISYNodeBeepButtonEntity] = []
+    uuid = isy.configuration[ISY_CONF_UUID]
+    entities: list[
+        ISYNodeQueryButtonEntity
+        | ISYNodeBeepButtonEntity
+        | ISYNetworkResourceButtonEntity
+    ] = []
     for node in hass_isy_data[ISY994_NODES][Platform.BUTTON]:
```

Comment:
It would be slightly cleaner to unpack `nodes` first 

`nodes = hass_isy_data[ISY994_NODES]`

Reply:
Revised